feat: hypercore page.

billy-the-fish opened the pull request:
Overview page explaining the advantages of the Hyperstore

billy-the-fish committed:
feat: hyperstore page.

billy-the-fish committed:
feat: hyperstore page.

billy-the-fish committed:
Merge branch 'latest' into 231-docs-rfc-create-a-single-page-document-explaining-why-hyperstore-is-great-and-hyperstore-crud

billy-the-fish committed:
feat: make link with compression more obvious.

billy-the-fish committed:
feat: make link with compression more obvious.

billy-the-fish committed:
Merge branch 'latest' into 231-docs-rfc-create-a-single-page-document-explaining-why-hyperstore-is-great-and-hyperstore-crud

ramonguiu committed:
Update hyperstore.md

Hyperstore -> hypercore

Signed-off-by: Ramon Guiu <ramon@timescale.com>

ramonguiu committed:
Rename hyperstore.md to hypercore.md

Signed-off-by: Ramon Guiu <ramon@timescale.com>

ramonguiu committed:
Update page-index.js

hyperstore -> hypercore

Signed-off-by: Ramon Guiu <ramon@timescale.com>

billy-the-fish committed:
Merge branch 'latest' into 231-docs-rfc-create-a-single-page-document-explaining-why-hyperstore-is-great-and-hyperstore-crud

billy-the-fish commented on use-timescale/hypercore.md:
```diff
@@ -0,0 +1,45 @@
+title: Hypercore
+excerpt: The Timescale hybrid row-columnar storage engine for real-time analytics, powered by time-series data
+products: [cloud,]
+keywords: [hyperscore, hypertable, compression, row-columnar storage, hyperstore]
```
```suggestion
keywords: [hyperscore, hypertable, compression, row-columnar storage, hypercore]
```

billy-the-fish committed:
Update use-timescale/hypercore.md

Signed-off-by: Iain Cox <iain@timescale.com>

billy-the-fish committed:
feat: add hyperstore back.

billy-the-fish committed:
chore: update troubleshooting info for DISTINCT. (#3461)

* chore: update troubleshooting info for DISTINCT.

billy-the-fish committed:
fix: revert structure. (#3471)

* fix: revert structure.

billy-the-fish committed:
fix: revert structure. (#3471)

* fix: revert structure.

atovpeko commented on use-timescale/hypercore.md:
```diff
@@ -0,0 +1,45 @@
+Hypercore’s hybrid approach combines the benefits of row-oriented and column-oriented formats 
+in each Timescale Cloud service:
+
+- **Fast ingest with Rowstore**: new data is initially written to the rowstore, it is optimized for 
```
Why capitalize Rowstore and Columnstore?

atovpeko replied:
```suggestion
- **Fast ingest with Rowstore**: new data is initially written to the rowstore, which is optimized for 
```

billy-the-fish committed:
Update use-timescale/hypercore.md

Co-authored-by: atovpeko <114177030+atovpeko@users.noreply.github.com>
Signed-off-by: Iain Cox <iain@timescale.com>

billy-the-fish committed:
chore: update on review.

billy-the-fish committed:
chore: merge updated folder structure.

esclymeTS committed:
Update changelog.md (#3481)

* Update changelog.md

---------

Signed-off-by: Eric Clymer <eric@timescale.com>
Signed-off-by: Iain Cox <iain@timescale.com>

billy-the-fish committed:
chore: update explanation for modes in Timescale Console. (#3474)

* chore: update explanation for modes in Timescale Console.

Co-authored-by: atovpeko <114177030+atovpeko@users.noreply.github.com>
Co-authored-by: Rahil Sondhi <rahilsondhi@gmail.com>

billy-the-fish committed:
chore: update docs with info about multiple CIDR blocks and changes t… (#3479)

* chore: update docs with info about multiple CIDR blocks, changes to pricing, and revalidate the procedure.

atovpeko committed:
Update PostgreSQL version (#3477)

Co-authored-by: Iain Cox <iain@timescale.com>

atovpeko committed:
foreign key constraint update (#3492)

Co-authored-by: Iain Cox <iain@timescale.com>

VineethReddy02 committed:
Remove live-migration reference to docker image URL (#3493)

* Remove reference to docker website

The live-migration documentation serves as our primary resource, with the first line highlighting live-migration.

Additionally, since there's no specific content for live-migration on Docker Hub, we should avoid referencing the Docker image page.

Signed-off-by: Vineeth Pothulapati <vineethpothulapati@outlook.com>

* Update live-migration.md

Signed-off-by: Vineeth Pothulapati <vineethpothulapati@outlook.com>

---------

Signed-off-by: Vineeth Pothulapati <vineethpothulapati@outlook.com>

atovpeko committed:
Timestamp vs timestamptz explanation (#3491)

Clarified the use of timestamp/timestamptz for partitioning hypertables

billy-the-fish committed:
fix: automatic compression compresses chunks automatically :-). (#3489)

billy-the-fish committed:
chore: rebase.

billy-the-fish committed:
chore: rebase.

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/signed)](https://cla-assistant.io/timescale/docs?pullRequest=3445) <br/>All committers have signed the CLA.

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/timescale/docs?pullRequest=3445) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://cla-assistant.io/timescale/docs?pullRequest=3445) before we can accept your contribution.<br/>**4** out of **5** committers have signed the CLA.<br/><br/>:white_check_mark: billy-the-fish<br/>:white_check_mark: ramonguiu<br/>:white_check_mark: atovpeko<br/>:white_check_mark: VineethReddy02<br/>:x: esclymeTS<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/timescale/docs?pullRequest=3445) it.</sub>

billy-the-fish committed:
Merge branch 'latest' into 231-docs-rfc-create-a-single-page-document-explaining-why-hyperstore-is-great-and-hyperstore-crud

Signed-off-by: Iain Cox <iain@timescale.com>

billy-the-fish commented:
Hi @ramonguiu , are we good to publish this or would you like to hold back a bit.

billy-the-fish commented:
I've moved the content from here to https://github.com/timescale/docs/pull/3642 so this is all wrapped up with API updates.
